config.py

```diff
@@ -3,68 +3,97 @@
 Config file
 ===========
 
-Configuration parameters for the study.
+Set the configuration parameters for the study.
 """
 
+import importlib
 import os
 from collections import defaultdict
+
+from bids import BIDSLayout
+
 import numpy as np
-from mne.datasets import sample
 
-# ``plot``  : boolean
-#   If True, the scripts will generate plots.
-#   If running the scripts from a notebook or spyder
-#   run %matplotlib qt in the command line to get the plots in extra windows
 
-plot = False
 
 ###############################################################################
 # DIRECTORIES
 # -----------
 #
-# ``study_path`` : str
-#    Set the `study path`` where the data is stored on your system.
+# ``bids_root`` : str
+#    Path to the directory that contains the study data in BIDS format.
 #
 # Example
 # ~~~~~~~
-# >>> study_path = '../MNE-sample-data/'
-# or
-# >>> study_path = '/Users/sophie/repos/ExampleData/'
+# >>> bids_root = '/home/my_user/my_bids_data/'
+
+if "MNE_BIDS_STUDY_CONFIG" in os.environ:
+    bids_root = '/home/stefanappelhoff/Downloads/eeg_matchingpennies'
+else:
+    # bids_root = '/Users/alex/work/data/osfstorage/eeg_matchingpennies'
+    bids_root = '/Users/alex/Dropbox/somato_recon/BIDS_complete/MNE-somato-data'  # noqa: E501
+
+# Make a pybids layout to easily get subjects, tasks, and other BIDS params
+layout = BIDSLayout(bids_root)
 
-study_path = sample.data_path()
 
 # ``subjects_dir`` : str
-#   The ``subjects_dir`` contains the MRI files for all subjects.
+#   Path to the directory that contains the MRI data files and their
+#   derivativesfor all subjects. Specifically, the ``subjects_dir`` is the
+#   $SUBJECTS_DIR used by the Freesurfer software.
 
-subjects_dir = os.path.join(study_path, 'subjects')
+subjects_dir = os.path.join(bids_root, 'derivatives', 'freesurfer', 'subjects')
 
-# ``meg_dir`` : str
-#   The ``meg_dir`` contains the MEG data in subfolders
-#   named my_study_path/MEG/my_subject/
 
-meg_dir = os.path.join(study_path, 'MEG')
+# ``plot``  : boolean
+#   If True, the scripts will generate plots.
+#   If running the scripts from a notebook or spyder
+#   run %matplotlib qt in the command line to get the plots in extra windows
 
 
-###############################################################################
-# SUBJECTS / RUNS
-# ---------------
-#
-# ``study_name`` : str
-#   This is the name of your experiment.
-#
-# Example
-# ~~~~~~~
-# >>> study_name = 'MNE-sample'
+plot = True
+plot = False
+
+
+# ``kind`` : str
+#   The kind of data to be analyzed, can be 'eeg', or 'meg'.
+
+# kind = 'eeg'
+kind = 'meg'
```

Comment:
you can infer this also from the data structure?

Say something like: "Found one MEG dataset with X subjects and Y tasks. Proceeding to analyze ..."

Reply:
yes, also with pybids :)

```Python
In [6]: layout.get(return_type='id', target='datatype')                                               
Out[6]: ['eeg']

```

Reply:
You probably make a small function for this pattern in `mne-bids` later. But let's make it work first!